tests/classes_private_fields/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,175 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`private_fields.js 1`] = `
+class A { #x; #y; }
+class B { #x = 0; #y = 1; }
+
+class C {
+  static #x;
+  static #y = 1;
+}
+
+class D {
+  #x
+  #y
+}
+
+class Point {
+  #x = 1;
+  #y = 2;
+
+  constructor(x = 0, y = 0) {
+    this.#x = +x;
+    this.#y = +y;
+  }
+
+  get x() { return this.#x }
+  set x(value) { this.#x = +value }
+
+  get y() { return this.#y }
+  set y(value) { this.#y = +value }
+
+  equals(p) { return this.#x === p.#x && this.#y === p.#y }
+
+  toString() { return \`Point<\${ this.#x },\${ this.#y }>\` }
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+class A {
+  #x;
+  #y;
+}
+class B {
+  #x = 0;
+  #y = 1;
+}
+
+class C {
+  static #x;
+  static #y = 1;
+}
+
+class D {
+  #x;
+  #y;
+}
+
+class Point {
+  #x = 1;
+  #y = 2;
+
+  constructor(x = 0, y = 0) {
+    this.#x = +x;
+    this.#y = +y;
```

Comment:
unrelated but the syntax looks really weird.

Reply:
Yeah... this proposal has a lot of history behind it 😓 https://github.com/tc39/proposal-private-fields/issues/14

Reply:
That issue isn't the half of it - I have some meeting notes from *1999* exploring how we might add private fields. Turns out it's hard; it's a miracle we figured out a way of doing it at all in a language without static types.